### PR TITLE
refactor: convert storybook apps to use strict mode

### PR DESCRIPTION
**Pull request by Blackbaud-CoreyArcher**
[AB#2325786](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2325786)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/878/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/tabs-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/878/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/878/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/878/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/878/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/878/playground/)

File: apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts [hunk 1]
```diff
@@ -32,18 +37,18 @@ export class WizardModalComponent implements OnInit {
...
+  // public onNextClick(): void {
+  //   this.activeIndex++;
+  // }
 
-  public onPrevClick(): void {
-    this.activeIndex--;
-  }
+  // public onPrevClick(): void {
+  //   this.activeIndex--;
+  // }
```
Comment (Blackbaud-SteveBrush) [inline apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts]: ☝️ ?

File: apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts [hunk 2]
```diff
@@ -32,18 +37,18 @@ export class WizardModalComponent implements OnInit {
   }
 
   public checkRequirementsMet(): void {
-    this.step2Disabled = !this.myForm.get('requiredValue1')?.value;
-    this.step3Disabled = !this.myForm.get('requiredValue2')?.value;
-    this.saveDisabled = this.myForm.get('requiredValue3')?.value;
+    this.step2Disabled = !this.myForm?.get('requiredValue1')?.value;
```
Comment (Blackbaud-SteveBrush) [inline apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts]: You can create `myForm` in the constructor and save yourself that first `?`.
```
public myForm: UntypedFormGroup;

constructor(formBuilder: UntypedFormBuilder) {
  this.myForm = formBuilder.group({...
}

// ...

this.step2Disabled = !this.myForm.get('requiredValue1')?.value;
```

File: apps/e2e/pages-storybook/src/app/action-hub/settings-modal.component.ts [hunk 3]
```diff
@@ -10,18 +10,21 @@ export class SettingsModalComponent {
...
+  #fb: UntypedFormBuilder;
+
   constructor(
-    private fb: UntypedFormBuilder,
+    fb: UntypedFormBuilder,
     public modal: SkyModalInstance,
     @Inject('modalTitle') public title: string
   ) {
-    const controls = {};
+    this.#fb = fb;
```
Comment (Blackbaud-SteveBrush) [inline apps/e2e/pages-storybook/src/app/action-hub/settings-modal.component.ts]: Do you still need this saved to a hash property? Could you remove `#fb` altogether?
```
this.form = fb.group(controls);
```

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Looks like some of the visual failures are valid?

File: apps/e2e/pages-storybook/src/app/action-hub/settings-modal.component.ts (hunk 3 above)
Comment (Blackbaud-CoreyArcher) [reply apps/e2e/pages-storybook/src/app/action-hub/settings-modal.component.ts]: changed

File: apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts (hunk 2 above)
Comment (Blackbaud-CoreyArcher) [reply apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts]: changed

File: apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts (hunk 1 above)
Comment (Blackbaud-CoreyArcher) [reply apps/e2e/tabs-storybook/src/app/wizard/wizard-modal.component.ts]: removed

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**